assets/ui/components/ArticleBodyHtml.jsx

```diff
@@ -66,6 +70,43 @@ class ArticleBodyHtml extends React.PureComponent {
         }
     }
 
+    executeScripts() {
+        const tree = this.bodyRef.current;
+        const loaded = [];
+
+        if (tree == null) {
+            return;
+        }
+
+        tree.querySelectorAll('script').forEach((s) => {
+            if (s.hasAttribute('src') && !loaded.includes(s.getAttribute('src'))) {
+                let url = s.getAttribute('src');
+
+                loaded.push(url);
+
+                if (url.startsWith('http')) {
+                    // change https?:// to // so it uses schema of the client
+                    url = url.substring(url.indexOf(':') + 1);
+                }
+
+                const script = document.createElement('script');
+
+                script.src = url;
+                script.async = true;
+
+                script.onload = () => {
+                    document.body.removeChild(script);
+                };
+
+                script.onerrror = (error) => {
+                    throw new URIError('The script ' + error.target.src + 'didn\'t load.');
```

Comment:
It might be good to include `error` in the exception raised, to make debugging easier

Reply:
you will see the error in the network tab

Reply:
Hah, good point